exchanges/poloniex/poloniex_websocket.go

```diff
@@ -28,1126 +27,561 @@ import (
 )
 
 const (
-	poloniexWebsocketAddress = "wss://api2.poloniex.com"
-	wsAccountNotificationID  = 1000
-	wsTickerDataID           = 1002
-	ws24HourExchangeVolumeID = 1003
-	wsHeartbeat              = 1010
+	poloniexWebsocketAddress        = "wss://ws.poloniex.com/ws/public"
+	poloniexPrivateWebsocketAddress = "wss://ws.poloniex.com/ws/private"
+
+	cnlExchange = "exchange"
+	cnlAuth     = "auth"
+
+	// Public channels
+	cnlSymbols    = "symbols"
+	cnlCurrencies = "currencies"
+	cnlCandles    = "candles"
+	cnlTrades     = "trades"
+	cnlTicker     = "ticker"
+	cnlBooks      = "book"
+	cnlBookLevel2 = "book_lv2"
+
+	// Authenticated channels
+	cnlOrders   = "orders"
+	cnlBalances = "balances"
+)
 
-	accountNotificationBalanceUpdate     = "b"
-	accountNotificationOrderUpdate       = "o"
-	accountNotificationPendingOrder      = "p"
-	accountNotificationOrderLimitCreated = "n"
-	accountNotificationTrades            = "t"
-	accountNotificationKilledOrder       = "k"
-	accountNotificationMarginPosition    = "m"
+var defaultSubscriptions = []string{
+	cnlCandles,
+	cnlTrades,
+	cnlTicker,
+	cnlBookLevel2,
+}
```

Comment:
🚧 Change request:
Please switch to subscription configuration and templating
```suggestion
var defaultSubscriptions = []string{
	cnlCandles,
	cnlTrades,
	cnlTicker,
	cnlBookLevel2,
}
```